src/main/java/uk/gov/hmcts/ccd/data/casedetails/CaseAuditEventMapper.java

```diff
@@ -39,11 +44,19 @@ public AuditEvent entityToModel(final CaseAuditEventEntity caseAuditEventEntity)
         auditEvent.setEventName(caseAuditEventEntity.getEventName());
         auditEvent.setSummary(caseAuditEventEntity.getSummary());
         auditEvent.setDescription(caseAuditEventEntity.getDescription());
+        if(caseAuditEventEntity.getSignificantItemEntity()!=null) {
```

Comment:
Formatting:
* Missing space after if
* Missing spaces on either side of !=

Reply:
addressed